### PR TITLE
NG-1169 ios

### PR DESCRIPTION
**Pull request by f1shear**

### CODE REVIEW

File: ios/Plugin/TwilioVideoViewController.swift [hunk 1]
```diff
@@ -58,6 +59,8 @@ class TwilioVideoViewController: UIViewController {
         self.disconnectButton.isHidden = true
         self.micButton.isHidden = true
 
+        self.statusLabel.text = "Connecting"
```
Comment (javipascual) [inline ios/Plugin/TwilioVideoViewController.swift]: not translated? if in android it is translated, it should be also here, if not, follow up task so we don't forget

File: ios/Plugin/TwilioVideoViewController.swift [hunk 2]
```diff
@@ -2,6 +2,17 @@ import UIKit
 import Foundation
 import TwilioVideo
 
+enum TwilioEvent: String {
```
Comment (javipascual) [inline ios/Plugin/TwilioVideoViewController.swift:5]: just curious, I suppose that Android uses the very same strings (I HOPE), so would it be possible to have some kind of common config for all supported platforms? does capacitor offer that?

**Review by javipascual — CHANGES_REQUESTED** [review]

File: ios/Plugin/Plugin.swift [hunk 3]
```diff
@@ -44,6 +46,38 @@ public class TwilioVideoPlugin: CAPPlugin {
...
+        if (options != nil) {
+            if let autoHideControls = options!["autoHideControls"] as? Bool {
+                self.viewController!.autoHideControls = autoHideControls
+            }
+
+            if let showSwitchCamera = options!["showSwitchCamera"] as? Bool {
+                self.viewController!.showSwitchCamera = showSwitchCamera
+            }
+
+            if let showSwitchCamera = options!["showSwitchCamera"] as? Bool {
```
Comment (javipascual) [inline ios/Plugin/Plugin.swift]: duplicated?

**Review by javipascual — CHANGES_REQUESTED** [review]

File: ios/Plugin/Plugin.swift (hunk 3 above)
Comment (f1shear) [reply ios/Plugin/Plugin.swift]: sorry, my bad

Comment (javipascual) [reply ios/Plugin/Plugin.swift]: nprob!

File: ios/Plugin/TwilioVideoViewController.swift (hunk 2 above)
Comment (f1shear) [reply ios/Plugin/TwilioVideoViewController.swift:5]: yes, they are the same strings. (I HOPE SO TOO). Capacitor doesn't offer that. However we may achieve this using a common json/xml file.

Comment (euclides-jr) [reply ios/Plugin/TwilioVideoViewController.swift:5]: It's not documented but Capacitor offers a help class to read values from capacitor.config.json which can be used as a key-value config store.

Comment (f1shear) [reply ios/Plugin/TwilioVideoViewController.swift:5]: we can do it in a follow up task to factor out common stuff

**Review by euclides-jr — APPROVED** [review]

File: ios/Plugin/Plugin.swift (hunk 3 above)
Comment (f1shear) [reply ios/Plugin/Plugin.swift]: fixed

File: ios/Plugin/TwilioVideoViewController.swift (hunk 2 above)
Comment (javipascual) [reply ios/Plugin/TwilioVideoViewController.swift:5]: great, yes!
https://media4care.atlassian.net/browse/NG-1241

**Review by javipascual — APPROVED** [review]

**Review by euclides-jr — APPROVED** [review]

**Pull request merged**